src/core/1/guides/essentials/installing-console/index.md

```diff
@@ -0,0 +1,48 @@
+---
+code: false
+type: page
+title: Kuzzle Admin Console
+order: 100
+---
+
+# Kuzzle Admin Console
+
+The Kuzzle Admin Console is a web application that lets you manage Kuzzle, including **data**, **real-time notifications** and **security**.
+
+If you don't want to install Kuzzle Admin Console yourself you can use our <a href="http://console.kuzzle.io/">publicly hosted</a> Kuzzle Admin Console. Otherwise, grab the source code [here](https://github.com/kuzzleio/kuzzle-admin-console/releases) and install it on your own environment.
+
+In both cases the configuration is the same and you'll be able to select which [Kuzzle](/core/1/guide/guides/essentials/installing-console/#connect-to-kuzzle) installation you want to manage.
+
+<div class="alert alert-info">
```

Comment:
Yes but there is 159 occurrence for this old alert div.. And its complicated to change because link inside these div are `<a href>` style instead of markdown style